boilerplate/index.ts

```diff
@@ -497,12 +493,27 @@ export function createBoilerplate(sdk: CDP) {
                             });
                         }
 
+
                         function createCloudStorageEvent(boilerplateEvent, remoteCloudStorageApplication) {
+                            // const conf = {};
+                            // for (const key of Object.keys(remoteCloudStorageApplication.configSchema.properties)) {
+                            //     conf[key] = 'value';
+                            // }
+
                             return bOps.applications.for(remoteCloudStorageApplication.id).dataevents.create({
                                 ...boilerplateEvent,
                                 schema: JSON.stringify(boilerplateEvent.schema),
-                                purposeIds: boilerplateEvent.purposeIds
-                            });
+                                purposeIds: boilerplateEvent.purposeIds,
+                                configValues: remoteCloudStorageApplication.name === 'Microsoft Azure Blob' ?
+                                    {
+                                        // these need to be explicitly defined and cannot be taken from the connector's or application's config schema since there is a ton of irrelevant info there
+                                        readContainer: "any container",
+                                        readFileNameRegex: null,
+                                        readFormat: null,
+                                    }
+                                    :
+                                    boilerplateEvent.configValues
```

Comment:
just a flavor comment we talked about before: short `if`, longer `else`  [means to swap if/else]